libphobos/m4/druntime/libraries.m4

```diff
@@ -16,6 +40,17 @@ AC_ARG_ENABLE(thread-lib,
 ])
 
 
+# DRUNTIME_LIBRARIES_TIME
+# -----------------------
++# Check whether -lrt is required for sched_yield, nanosleep, or clock_gettime.
+AC_DEFUN([DRUNTIME_LIBRARIES_TIME],
+[
+AC_SEARCH_LIBS([sched_yield], [rt])
+AC_SEARCH_LIBS([nanosleep], [rt])
+AC_SEARCH_LIBS([clock_gettime], [rt])
+])
+
```

Comment:
> Do we actually need this? If librt is required, the driver should still link it automatically as GCC will define TIME_LIBRARY. We could probably rewrite this to use gdc instead and simply assert that the functions are available

* core.thread.Thread.sleep() uses `nanosleep` - doesn't need `rt` on linux.
* core.time.MonoTime.currTime() uses `clock_gettime` - doesn't need `rt` on linux.
* core.thread.Thread.yield() uses `sched_yield` - doesn't need `rt` on linux.

I had a look around, but I couldn't find anything.  GCC backend will define `TIME_LIBRARY` if it is really required.  It's been so long ago I don't remember the reason why I added `rt` to be on by default.  But I'm not seeing any trouble with linking statically with it's removal (on Linux).